README.md

```diff
@@ -4,7 +4,7 @@
     <a href="https://developer.aleo.org"> <img alt="Website" src="https://img.shields.io/badge/Developer_Docs-online-blue"></a>
     <a href="https://circleci.com/gh/AleoHQ/aleo"><img src="https://circleci.com/gh/AleoHQ/sdk.svg?style=svg"></a>
     <a href="https://discord.gg/5v2ynrw2ds"><img src="https://img.shields.io/discord/700454073459015690?logo=discord"/></a>
-    <a href="https://allcontributors.org/"><img src="https://img.shields.io/github/all-contributors/AleoHQ/sdk?color=ee8449&style=flat-square"/></a>
+    <a href="https://GitHub.com/AleoHQ/sdk"><img src="https://img.shields.io/badge/contributors-23-ee8449"/></a>
```

Comment:
This would anchor it to the contributors section if you wanted.

```suggestion
    <a href="https://github.com/AleoHQ/sdk#%EF%B8%8F-contributors"><img src="https://img.shields.io/badge/contributors-23-ee8449"/></a>
```